src/account/AccountItem.js

```diff
@@ -63,7 +64,9 @@ export default class AccountItem extends PureComponent<Props> {
             <RawLabel style={styles.text} text={email} numberOfLines={1} />
             <RawLabel style={styles.text} text={realm} numberOfLines={1} />
           </View>
-          {!showDoneIcon ? (
+          {showProgress ? (
```

Comment:
I am usually fine with a single ternary operator, but two of them usually make the logic too messy.

Reply:
Moved this logic to `AccountList`